lib/src/util/number.dart

```diff
@@ -30,6 +30,17 @@ bool fuzzyEquals(num number1, num number2) {
           (number2 * _inverseEpsilon).round();
 }
 
+/// Like [fuzzyEquals], but allows null values for [number1] and [number2].
+///
+/// null values are only equal to one another.
+bool fuzzyEqualsNullable(num? number1, num? number2) {
+  if (number1 == number2) return true;
+  if (number1 == null || number2 == null) return false;
+  return (number1 - number2).abs() <= _epsilon &&
+      (number1 * _inverseEpsilon).round() ==
+          (number2 * _inverseEpsilon).round();
+}
```

Comment:
Nit: To avoid possible divergence, `fuzzyEqualsNullable` could call `fuzzyEquals` after line 38

Reply:
That would duplicate the `number1 == number2` test, which isn't super relevant but for code this low-level I'd like to keep it as efficient as possible since it's likely to be run very frequently.